tools/greenbone-nvt-sync.in

```diff
@@ -632,6 +635,17 @@ do_feedversion () {
   fi
 }
 
+create_feed_lock_file ()
+{
+  if [ -f $OPENVAS_RUN_DIR/feed-update.lock ] ; then
+    log_err "Another process related to the feed update is already running."
```

Comment:
```suggestion
    log_warning "Another process related to the feed update is already running."
```

I really do think `log_warning` would be more appropriate here. Being unable to create a lock file is not an error condition which generally prevents the script from running, but an expected, temporary condition. This would IMHO justify at best a warning.